lua/lspconfig/server_configurations/snyk_ls.lua

```diff
@@ -6,7 +6,9 @@ return {
     root_dir = util.root_pattern '.git',
     single_file_support = true,
     settings = {},
-    init_options = {},
+    init_options = {
+      activateSnykCode = "true",
```

Comment:
this boolean as string type to send ?

Reply:
this is how it is used in snyk's documentation,
<img width="687" alt="image" src="https://github.com/neovim/nvim-lspconfig/assets/41305372/53ca1370-5f9c-4c13-b19d-16d6f83876bc">

https://docs.snyk.io/integrate-with-snyk/use-snyk-in-your-ide/snyk-language-server

Reply:
thanks just fix lint